lib/new_relic/agent/instrumentation/active_merchant.rb

```diff
@@ -34,4 +34,19 @@ class ActiveMerchant::Billing::Gateway
       end
     end
   end
+
+  executes do
+    if ActiveMerchant::VERSION < Gem::Version.new('1.65.0')
+      deprecation_msg = 'The Ruby Agent is dropping support for ActiveMerchant versions below 1.65.0 ' \
```

Comment:
The code base has loads of "Ruby Agent" and "Ruby agent" instances all over the place, both in comments and in printed out content. No need to address it with this PR though, I'll create an issue.

Reply:
Created #1118